odoo/addons/test_read_group/tests/test_m2m_grouping.py

```diff
@@ -216,6 +216,26 @@ def test_complex_case(self):
             },
         ])
 
+    def test_order_by_one2many_id(self):
```

Comment:
```suggestion
    def test_order_by_many2one_id(self):
```
same for the next line